.jenkins/jenkins_pod.yml

```diff
@@ -1,33 +1,54 @@
+
 apiVersion: v1
 kind: Pod
 spec:
+  ##
   imagePullSecrets:
     - name: talend-registry
+  ##
   containers:
-    - name: main
+    ##
+    - name: docker-daemon # Container that hosts the Docker daemon/socket
+      image: artifactory.datapwn.com/docker-io-remote/docker:24.0.7-dind-rootless # Rootless versions
+      command:
+        - 'dockerd-entrypoint.sh' # Override entry point to add parameters
+      args:
+        - '--tls=false' # De-activate TLS (not possible in rootless mode)
+      env:
+        - name: DOCKER_TLS_CERTDIR  # Don't generate the certificate since it's not used (faster startup)
+          value: ""
+      securityContext:
+        privileged: true # Needed to bootstrap the Docker image. as rootless, it uses a non-root user at startup
+    ###
+    - name: main # default container for jenkins job execution
       # 3.0.9-20220930152032 is the last root version of custom-builder
-      image: 'artifactory.datapwn.com/tlnd-docker-dev/talend/common/tsbi/custom-builder:3.0.9-20220930152032'
+      # If you use this pod you will be jenkins user
+      # TSBI customBuilder documentation: https://github.com/Talend/tsbi-images/blob/main/tsbi/custom-builder/README.md
+      image: 'artifactory.datapwn.com/tlnd-docker-dev/talend/common/tsbi/custom-builder:4.0.24-20241016140323'
       command: [ cat ]
       tty: true
       volumeMounts: [
-        { name: efs-jenkins-component-runtime-m2, mountPath: /root/.m2/repository},
-        { name: efs-jenkins-connectors-asdf, mountPath: /root/.asdf/installs, subPath: installs }
+        { name: efs-jenkins-component-runtime-m2-rootless,
+          mountPath: /home/jenkins/.m2/ },
+        { name: efs-jenkins-connectors-asdf-rootless,
+          mountPath: /home/jenkins/.asdf/installs},
       ]
-      resources: {requests: {memory: 6G, cpu: '4.0'}, limits: {memory: 8G, cpu: '5.0'}}
+      resources: { requests: { memory: 4G, cpu: '2' } }
```

Comment:
It could, but I didn't feel it, and it will lower the time to have an available slot.
And due to our occupation of the cluster we probably always have the memory: 6G, cpu: '4.0' so not so far.
Time on master:   24mn
Time on my branch:  14mn seem better.

We will add resources if needed after